identity/credentials_webauthn.go

```diff
@@ -49,8 +57,16 @@ func (c CredentialsWebAuthn) ToWebAuthn() (result []webauthn.Credential) {
 }
 
 // PasswordlessOnly returns only passwordless credentials.
-func (c CredentialsWebAuthn) PasswordlessOnly() (result []webauthn.Credential) {
+func (c CredentialsWebAuthn) PasswordlessOnly(authenticatorResponseFlags *protocol.AuthenticatorFlags) (result []webauthn.Credential) {
 	for k, cc := range c {
+		// Upgrade path for legacy webauthn credentials. Only possible if we are handling a response from an authenticator.
+		if c[k].Flags == nil && authenticatorResponseFlags != nil {
+			c[k].Flags = &CredentialWebAuthnFlags{
```

Comment:
```suggestion
		if cc.Flags == nil && authenticatorResponseFlags != nil {
			cc.Flags = &CredentialWebAuthnFlags{
```

Reply:
I believe that then we would need to do `c[k]=cc` because `c` is a struct and not a pointer